### PR TITLE
Adding import for the compute_route resource

### PR DESCRIPTION
**Pull request by skalle**
Chipping away at the resource imports.

### CODE REVIEW

File: builtin/providers/google/import_compute_route_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	resourceName := "google_compute_network.foobar"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeRouteDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeRoute_basic,
+			}, {
```
Comment (grubernaut) [inline builtin/providers/google/import_compute_route_test.go]: Add a newline here after the first `resource.TestStep` object. 
```
},
{
```

File: builtin/providers/google/import_compute_route_test.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	resourceName := "google_compute_network.foobar"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeRouteDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeRoute_defaultInternetGateway,
+			}, {
```
Comment (grubernaut) [inline builtin/providers/google/import_compute_route_test.go]: Same here

**Comment by grubernaut** [comment]
Hi @skalle, thanks for the submission! 
Would you be able to comment the test output from running these two acceptance tests? 
Also would need to update the documentation for the `compute_route` resource noting that import functionality exists. Thanks!

**Comment by skalle** [comment]
Hey @grubernaut  , thx. for the review. 

Sorting out the format and documentation when back to work tomorrow.

**Comment by stack72** [comment]
Hi @skalle / @grubernaut 

Just a FYI, I ran the tests a for this and they pass as expected and made the small formatting changes. I have pushed this to master already

```
% make testacc TEST=./builtin/providers/google TESTARGS='-run=TestAccComputeRoute'
==> Checking that code complies with gofmt requirements...
go generate $(go list ./... | grep -v /terraform/vendor/)
2017/05/04 19:14:53 Generated command/internal_plugin_list.go
TF_ACC=1 go test ./builtin/providers/google -v -run=TestAccComputeRoute -timeout 120m
=== RUN   TestAccComputeRoute_importBasic
--- PASS: TestAccComputeRoute_importBasic (68.36s)
=== RUN   TestAccComputeRoute_importDefaultInternetGateway
--- PASS: TestAccComputeRoute_importDefaultInternetGateway (89.60s)
=== RUN   TestAccComputeRoute_basic
--- PASS: TestAccComputeRoute_basic (80.55s)
=== RUN   TestAccComputeRoute_defaultInternetGateway
--- PASS: TestAccComputeRoute_defaultInternetGateway (84.01s)
PASS
ok  	github.com/hashicorp/terraform/builtin/providers/google	322.541s
```

Thanks

Paul

**Pull request merged**